econplayground/templates/assignment/step_detail.html

```diff
@@ -71,11 +71,11 @@ <h1>{% if object.name %}{{ object.name }}{% else %}Step {{ object.pk }}{% endif
             </div>
         {% endif %}
 
+        <h2>{{object.question.title}}</h2>
```

Comment:
This title and prompt display should remain within the `{% if object.question %}` conditional, because it will cause an exception if there is no question.